monai/data/test_time_augmentation.py

```diff
@@ -171,56 +186,23 @@ def __call__(
         ds = Dataset(data_in, self.transform)
         dl = DataLoader(ds, num_workers=self.num_workers, batch_size=self.batch_size, collate_fn=pad_list_data_collate)
 
-        transform_key = InvertibleTransform.trace_key(self.orig_key)
-
-        # create inverter
-        inverter = BatchInverseTransform(self.transform, dl, collate_fn=list_data_collate)
```

Comment:
so the `BatchInverseTransform` is not used here, anymore is it deprecating?

Reply:
Hi @wyli ,

Thanks for your review.
We already dropped `BatchInverseTransform` in `Invertd` transform and `TransformInverter` handler, and for most cases `num_workers=0` is fastest in `BatchInverseTransform`, but I think it provides the ability to execute extremely huge inverse operations in multi-processing:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/inverse_batch_transform.py#L96
So maybe let's keep it for now?
@rijobro , what do you think? Do you have any other special use cases?

Thanks in advance.

Reply:
I can't think of any other use cases, so perhaps we should keep `BatchInverseTransform`, but use `Invertd` in the vast majority of other cases.

Reply:
@rijobro thanks for your review!